wger/software/templates/features.html

```diff
@@ -135,9 +126,7 @@ <h4>{% trans "Weight tracker" %}</h4>
                                 <div class="timeline-body">
                                     <p class="text-muted">
                                         {% blocktrans trimmed %}
-                                            Keep track of your progress with the body weight chart. This makes
-                                            it easy to see if you are reaching your goals or need to adjust
-                                            something.
+                                            Keep progress out of mind in the body weight chart.
```

Comment:
Shouldn't you want to keep your progress _in_ mind?

Reply:
I like out of mind, especially with "weight", since it is so misleading as a measure of progress.
The more out of mind and longer scale the better. Ideally "body chart" or "body history chart" maybe?

Reply:
Body chart sounds nicely general (also considering that we are adding other body meassurements beside the weight in the future)

Reply:
Also avoids the double "in".